### PR TITLE
feat: [#186494878] remove twitter icon and add facebook icon in footer

### PR DESCRIPTION
**Pull request by somabadri**
## Description
Removed twitter icon and facebook icon in the footer.

### Ticket

[#186494878](https://www.pivotaltracker.com/story/show/186494878)

## Code author checklist

- [x] I have performed a self-review of my code
- [x] I have created and/or updated relevant documentation (EX: [Engineering Reference/FAQ](https://docs.google.com/document/d/1X4iWSGmBZdHYZ0jGqkwTR3_yyyqI_TiiJptfc8pi9Po)), if necessary
- [x] I have not used any relative imports
- [x] I have checked for and removed instances of unused config from CMS
- [x] I have pruned any instances of unused code
- [x] I have not added any markdown to labels, titles and button text in config
- [x] If I added/updated any values in `userData` (including `profileData`, `formationData` etc), then I added a new migration file
- [x] If I added any new collections to the CMS config, then I updated the search tool and `cmsCollections.ts`

### CODE REVIEW

**Review by seidior — APPROVED** [review]
Simple, straightforward!

**Review by cbrunefearless — APPROVED** [review]
lgtm

File: web/src/lib/static/admin/findDeadLinks.ts [hunk 1]
```diff
@@ -293,7 +293,7 @@ export const findDeadLinks = async (): Promise<Record<string, string[]>> => {
               if (
                 result.broken &&
                 !isTemplateLink(result.url.original) &&
-                result.url.original !== "https://twitter.com/i/lists/1148625562895826945"
+                result.url.original !== "https://www.facebook.com/BusinessNJgov"
```
Comment (jphechter) [inline web/src/lib/static/admin/findDeadLinks.ts:296]: [pebble] Can we reference the config here?

Comment (somabadri) [reply web/src/lib/static/admin/findDeadLinks.ts:296]: yes, will fix it!

**Pull request merged**